readme.md

```diff
@@ -779,6 +779,11 @@ _Novels which emphasize adventure, set mainly or entirely in outer space, usuall
 
 > The trilogy is set in a universe with a wealth of worlds and artificial orbiting colonies. The plot is centered on the souls of the dead coming back from a hellish “beyond” to possess the living, and the latter fighting back. It was followed by a companion to the series, _The Confederation Handbook_, an informational book containing data about the universe of the _Night’s Dawn_ trilogy. Hamilton re-set several earlier short stories in the Confederation timeline, published as the collection _A Second Chance at Eden_, including the newly written title novella.
 
+#### [Revelation Space](https://www.goodreads.com/series/56392-revelation-space) (2000-2018) *by [https://en.wikipedia.org/wiki/Alastair_Reynolds)* [4.0 avg]
+
+Alastair Reynolds take on the Fermi Paradox is a vast and brutal epic, consisting of the three "main" books (forming the "Inhibitor Trilogy") and a a few other books and novellas set in the same universe. Revelation Space is "hard space opera". - [@nahkampf](https://github.com/nahkampf)
```

Comment:
Duplicate `a a`, and a missing apostrophe after `Reynolds`. 

And - can you add a little on why this should be here? Why are you suggesting it? What did you think?